lib/std/fs.zig

```diff
@@ -2139,7 +2137,7 @@ pub const Dir = struct {
         defer atomic_file.deinit();
 
         try copy_file(in_file.handle, atomic_file.file.handle);
-        return atomic_file.finish();
+        return try atomic_file.finish();
```

Comment:
why is that necessary? it seems like a nop change to me... either way, wouldn't instead replacing `return` with `try` be more appropriate?, e.g.
```suggestion
        try atomic_file.finish();
```

Reply:
it makes it more clear to the reader who may or may not be at an editor that the function can error, before i thought it did not and it confused me

Reply:
ok, but as the function otherwise returns void, it makes more sense to just replace `return` with `try` instead of using both.